### PR TITLE
Remove automatic server-side prints

### PR DESCRIPTION
**Pull request by hasLeland**
Remove print statements which are automatically run by the server, so errors and such will *not* print to the stdout of the server, instead they'll be included in the response.

### CODE REVIEW

File: gengokit/httptransport/templates/server.go [hunk 1]
```diff
@@ -262,4 +250,16 @@ func headersToContext(ctx context.Context, r *http.Request) context.Context {
...
+	return func(ctx context.Context, r *http.Request) (interface{}, error) {
+		logger.Log("method", r.Method, "url", r.URL.String())
+		rv, err := next(ctx, r)
+		if err != nil {
+			logger.Log("method", r.Method, "url", r.URL.String(), "Error", err)
+		}
+		return rv, err
+	}
+}
+
```
Comment (adamryman) [inline gengokit/httptransport/templates/server.go]: Trailing whitespace

**Review by adamryman — APPROVED** [review]
One nit. Perfect otherwise!

File: gengokit/httptransport/templates/server.go (hunk 1 above)
Comment (hasLeland) [reply gengokit/httptransport/templates/server.go]: Done!

**Review by zaquestion — APPROVED** [review]

File: gengokit/httptransport/templates/server.go [hunk 2]
```diff
@@ -262,4 +250,15 @@ func headersToContext(ctx context.Context, r *http.Request) context.Context {
 
 	return ctx
 }
+
+func HTTPDecodeLogger(next httptransport.DecodeRequestFunc, logger log.Logger) httptransport.DecodeRequestFunc {
```
Comment (zaquestion) [inline gengokit/httptransport/templates/server.go:254]: Any reason for keeping?

Comment (hasLeland) [reply gengokit/httptransport/templates/server.go:254]: Basically, as a "just in case", but that may or may not be valid. Think it should be deleted?

**Pull request merged**